l10n_es_aeat_sii_oca/wizards/account_move_reversal.py

```diff
@@ -12,7 +12,14 @@ def _default_sii_refund_type_required(self):
         invoices = self.env["account.move"].browse(
             self.env.context.get("active_ids"),
         )
-        return any(invoices.mapped("company_id.sii_enabled"))
+        # If any of the invoices part of the active_ids match the criteria,
+        # show the field
+        return bool(
+            invoices.filtered(
+                lambda i: i.type in ("in_invoice", "out_invoice")
```

Comment:
In v14 it's `move_type`

Reply:
Sorry again :man_facepalming: 

Changes made :+1: